lib/src/models/data/schedule.dart

```diff
@@ -73,10 +72,22 @@ class ScheduleModel extends Model {
 					day == null ? null : Day.fromJson(day)
 			]
 		];
+
 		setToday();
 		notifyListeners();
 	}
 
+	///
+	Future<Map<String, Schedule>> getDefaultSchedules() async {
+		for(MapEntry<String, String> entry in await
+		(Services.instance.database.calendar.cloud.getDefaultSchedules()).entries){
+			for(Schedule schedule in Schedule.schedules){
+				if(schedule.name == entry as String) {
```

Comment:
```suggestion
				if (schedule.name == entry) {
```

You shouldn't have to use `as`, it's mostly for when you don't know the type of a variable and using it suppresses any type errors. Useful in some cases, but dangerous in most.